api/src/wfl/service/cromwell.clj

```diff
@@ -18,9 +18,11 @@
 (def active-statuses
   "The statuses an active Cromwell workflow can have."
   ["Aborting"
-   "On Hold"
    "Running"
-   "Submitted"])
+   "Submitted"
+   "Queued",
+   "Launching",
+   "Unknown"])
```

Comment:
I'm not sure if we want to update them in-place, since WFL needs to support both Cromwell and Terra. From what I can tell, the new set of statuses are still specific to Terra?

Reply:
I added the statuses from the rawls swagger into the rawls service namespace, so now there are statuses in cromwell and rawls.